sim/deathknight/bone_shield.go

```diff
@@ -44,13 +45,16 @@ func (dk *Deathknight) registerBoneShieldSpell() {
 		},
 	})
 
+	baseCost := float64(core.NewRuneCost(10, 0, 0, 1, 0))
```

Comment:
Bone shield does not cost 10 runic power or I'm not understanding this. isnt it supposed to be -10 because it gives rp